backend/parseport/http_client.py

```diff
@@ -0,0 +1,4 @@
+import urllib3
+
+
+http_client = urllib3.PoolManager()
```

Comment:
I've outfactored this to a separate file so we don't create a new one for each app.